examples/hoomd-blue/umbrella_integration/integration.py

```diff
@@ -13,28 +13,30 @@
 from pysages.methods import UmbrellaIntegration
 
 
-param1 = {"A": 0.5, "w": 0.2, "p": 2}
+params = {"A": 0.5, "w": 0.2, "p": 2}
 
 
-def generate_context(**kwargs):
+def generate_context(A=0.5, w=0.2, p=2, **kwargs):
```

Comment:
Are we using these anywhere?
We are pulling from `params` right now.

Reply:
We don't need these I added them just to realize they weren't necessary, but forgot to remove them.

Reply:
we should remove them then